src/tests/functional/plugin/shared/include/behavior/ov_executable_network/get_metric.hpp

```diff
@@ -178,6 +178,31 @@ TEST_P(OVClassExecutableNetworkGetMetricTest_DEVICE_PRIORITY, GetMetricNoThrow)
     ASSERT_EQ(value, configuration[ov::device::priorities.name()].as<std::string>());
 }
 
+TEST_P(OVClassExecutableNetworkGetMetricTest_DEVICE_PROPERTIES, GetMetricWithDevicePropertiesNoThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device, configuration);
+    int32_t expected_value = configuration[device_name].as<ov::AnyMap>()[ov::num_streams.name()].as<int32_t>();
+    int32_t actual_value = -1;
+    ASSERT_NO_THROW(actual_value = compiled_model.get_property(ov::device::properties(device_name, ov::num_streams)));
+    ASSERT_EQ(expected_value, actual_value);
+}
+
+TEST_P(OVClassExecutableNetworkGetMetricTestUnsupportConfigThrow_DEVICE_PROPERTIES,
+       GetMetricWithDevicePropertiesThrow) {
+    ov::Core ie = createCoreWithTemplate();
+    auto compiled_model = ie.compile_model(simpleNetwork, target_device, configuration);
+    // throw exception when getting unsupported property through device's executable network via this API
+    ASSERT_THROW(compiled_model.get_property(ov::device::properties(device_name, ov::device::priorities)),
+                 ov::Exception);
+}
+
+TEST_P(OVClassExecutableNetworkGetMetricTestInvalidDeviceThrow_DEVICE_PROPERTIES, GetMetricWithDevicePropertiesThrow) {
+    ov::Core ie = createCoreWithTemplate();
```

Comment:
Will it work if we call:
`ie.get_property(ov::device::properties...)` ?

Reply:
will not work because it hasn't been implemented to support like this passing `ov::device::properties(...)` to get property via IE.